utils/validate_images.js

```diff
@@ -27,31 +28,52 @@ const validateImageCounts = (quickstartDirs) => {
       const quickstartDirName = path.dirname(quickstart);
       // get all images for a quickstart
       const imagePaths = glob.sync(
-        path.resolve(path.dirname(quickstart), '**/*.+(png|jpeg|jpg|svg)')
+        path.resolve(quickstartDirName, '**/*.+(png|jpeg|jpg|svg)')
       );
       const quickstartConfig = readQuickstartFile(quickstart).contents[0];
       const iconPath = quickstartConfig.icon
-        ? path.resolve(path.dirname(quickstart), quickstartConfig.icon)
+        ? path.resolve(quickstartDirName, quickstartConfig.icon)
         : null;
       const logoPath = quickstartConfig.logo
-        ? path.resolve(path.dirname(quickstart), quickstartConfig.logo)
+        ? path.resolve(quickstartDirName, quickstartConfig.logo)
         : null;
-
+      
+      // Max images is per dashboard so we need to account for this by getting the number of dashboards
+      const dashboardCount = glob.sync(
+        path.resolve(quickstartDirName, 'dashboards/*.json')
+      ).length;
+      
       const screenshotPaths = imagePaths.filter(
-        (p) => p !== iconPath && p !== logoPath
+        (p) => p !== iconPath && p !== logoPath && !p.includes(DASHBOARD_IMAGES_PATH)
       );
 
-      if (screenshotPaths.length > MAX_NUM_IMG) {
+      const dashboardImagePaths = imagePaths.filter(
+        (p) => p !== iconPath && p !== logoPath && p.includes(DASHBOARD_IMAGES_PATH)
+      );
+      
+      // Each dashboard is allowed MAX_NUM_IMG dashboards
+      if (screenshotPaths.length > (MAX_NUM_IMG * dashboardCount)) {
         return {
           folder: quickstartDirName,
+          dashboardCount,
           imageCount: screenshotPaths.length,
+          maxImages: MAX_NUM_IMG * dashboardCount,
+        };
+      }
+
+      if (dashboardImagePaths.length > (MAX_NUM_IMG * dashboardCount)) {
+        return {
+          folder: quickstartDirName + DASHBOARD_IMAGES_PATH,
+          dashboardCount,
+          imageCount: dashboardImagePaths.length,
+          maxImages: MAX_NUM_IMG * dashboardCount,
         };
       }
     })
     .filter(Boolean);
 
   if (directories.length) {
-    core.setFailed('Components should contain less than 6 images');
+    core.setFailed('Components should contain less than 6 images each');
```

Comment:
We probably want a unique message for screenshots and images, I don't think users would be able to tell if they had a problem with images or screenshots from this message.